doc/api/permissions.md

```diff
@@ -440,7 +446,154 @@ not adopt the origin of the `blob:` URL.
+
+* `permission.deny(scope [,parameters])`
+
+API Call to _deny_ permissions at runtime. e.g
+
+```js
+process.permission.deny('fs'); // Deny permissions to ALL fs operations
+
+// Deny permissions to ALL FileSystemOut operations
+process.permission.deny('fs.write');
+// deny FileSystemOut permissions to the protected-folder
+process.permission.deny('fs.write', '/home/rafaelgss/protected-folder');
+// Deny permissions to ALL FileSystemIn operations
+process.permission.deny('fs.read');
+// deny FileSystemIn permissions to the protected-folder
+process.permission.deny('fs.read', '/home/rafaelgss/protected-folder');
+```
+
+It returns a boolean as an operation result.
+
+* `permission.check(scope [,parameters])`
+
+API Call to _check_ permissions at runtime. e.g:
+
+```js
+process.permission.check('fs.write'); // true
+process.permission.check('fs.write', '/home/rafaelgss/protected-folder'); // true
+
+process.permission.deny('fs.write', '/home/rafaelgss/protected-folder');
+
+process.permission.check('fs.write'); // true
+process.permission.check('fs.write', '/home/rafaelgss/protected-folder'); // false
+```
+
+#### File System Permissions
+
+To allow access to the filesystem, you need to use the flag `--alow-fs`
+
+```console
+$ node --experimental-permission --allow-fs=read index.js
+Hello world!
+(node:19836) ExperimentalWarning: Permission is an experimental feature
+(Use `node --trace-warnings ...` to show where the warning was created)
+```
+
+The valid arguments for the `--allow-fs` flag option are:
+
+* `write` - To manage `FileSystemOut` (Writing) operations.
+* `read` - To manage `FileSystemIn` (Reading) operations.
+* `fs` - To allow all the `FileSystem` operations.
+
+Example:
+
+* `--allow-fs=read:/tmp/` - It will allow `FileSystemIn` access to the `/tmp/`
+  folder.
+* `--allow-fs=read:/tmp/:/home/.gitignore` - It allows `FileSystemIn` access to
+  the `/tmp/` folder **and** the `/home/.gitignore` path.
+
+You can also mix both arguments:
+
+* `--allow-fs=write,read:/tmp/` - It will allow `FileSystemIn` access to the
+  `/tmp/` folder **and** allow **all** the `FileSystemOut` operations.
+* **Note**: It accepts wildcard parameters as well.
+  For instance: `--allow-fs=write:/home/test*` will allow everything that
+  matches the wildcard. e.g: `/home/test/file1` or `/home/test2`
+
+There are constraints you need to know before using this system:
+
+* Native modules are not supported by this mechanism.
+* Relative paths aren't supported yet through the CLI (`--allow-fs`).
+  The runtime API supports relative paths.
+* File Descriptors opened before a permission set is applied won't be
+  validated. Consider the following snippet:
+
+```js
+const fs = require('fs');
+
+// Open a fd
+const fd = fs.openSync('./README.md', 'r');
+// Then, deny access to all fs.read operations
+process.permission.deny('fs.read');
+// This call will NOT fail and the file will be read
+const data = fs.readFileSync(fd);
+```
```

Comment:
````suggestion
  ```js
  const fs = require('node:fs');

  // Open a fd
  const fd = fs.openSync('./README.md', 'r');
  // Then, deny access to all fs.read operations
  process.permission.deny('fs.read');
  // This call will NOT fail and the file will be read
  const data = fs.readFileSync(fd);
  ```
````

Reply:
Why whitespaces?